src/MonetizationInterface.php

```diff
@@ -218,4 +218,21 @@ public function formUserAdminPermissionsAlter(&$form, FormStateInterface $form_s
    */
   public function userRolePresave(RoleInterface $user_role);
 
+  /**
+   * Get URL for "Add Credit" link.
+   *
+   * Get the URL object for the "add credit" product in the requested currency.
+   * Fallback to the user's prepaid balances page if there isn't a product for
+   * the requested currency.
+   *
+   * @param string $currency_id
+   *   The currency ID.
+   * @param \Drupal\user\UserInterface $account
+   *   The user entity for which to generate the fallback link.
+   *
+   * @return \Drupal\Core\Url
+   *   The URL object.
+   */
+  public function getAddCreditUrl($currency_id, UserInterface $account);
```

Comment:
I'm not so keen on this one. I'm thinking the add credit module should be the one to alter to add information only it is aware of.